server/src/controllers/Restaurant.ts

```diff
@@ -45,7 +45,16 @@ export class RestaurantController {
   })
   @ResponseSchema(Restaurant)
   public async get(@Param('id') id: string) {
-    return Restaurant.findOne({ id });
+    return Restaurant.findOne(
```

Comment:
Just to clarify. The client only has to make one request to this endpoint, and it'll be given back a nested response with all information from sections, tables, and reservations.

Is there anything that is a part of the entities that is not returned in the response?

Reply:
Correct, it'll look roughly like
```
{
  sections: [
    {
      tables: []
    },
   {
      tables: []
    },
  ]
}
```

The things that will **not** be included are any other foreign keys such as `section.shifts` or `table.services`

Reply:
👍